.github/workflows/cd.yml

```diff
@@ -52,4 +58,9 @@ jobs:
         with:
           tag_name: ${{ steps.tag_version.outputs.new_tag }}
           release_name: Release ${{ steps.tag_version.outputs.new_tag }}
-          body: ${{ steps.tag_version.outputs.changelog }}
\ No newline at end of file
+          body: ${{ steps.tag_version.outputs.changelog }}
+
+      - name: Deploy to Ansible Galaxy
```

Comment:
So last step in the updated CD pipeline is to deploy to galaxy after updating the GH version - which ensures everything is always on latest and in sync between GH and Galaxy - right?